app/Jobs/GitHub/SyncRepositoryJob.php

```
@@ -0,0 +1,31 @@
+namespace App\Jobs\GitHub;
+
+use App\DTO\GitHub\RepositoryData;
+use App\Models\Organization;
+use App\Services\GitHub\GitHubSyncOrchestrator;
+
+class SyncRepositoryJob extends AbstractSyncJob
+{
+    public function __construct(
+        private int $organizationId,
+        private RepositoryData $repositoryDto,
+        private bool $force = false
```

PHPMD (2.15.0)

The method __construct has a boolean flag argument $force, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)